src/vault/client/ext/aws.clj

```diff
@@ -56,34 +61,39 @@
   ^AWS4Signer
   []
   (doto (AWS4Signer.)
-        (.setServiceName "sts")
-        (.setRegionName "us-east-1")))
+    (.setServiceName "sts")
+    (.setRegionName "us-east-1")))
+
+
+(defn- str->b64str
+  [^java.util.Base64$Encoder b64 ^String s]
```

Comment:
These are cheap enough to make (and you do few enough calls) that it would be cleaner to make a new one each call to this function.